apis/kueue/v1beta1/fairsharing_types.go

```diff
@@ -49,4 +53,31 @@ type FairSharingStatus struct {
 	// weight of zero and is borrowing, this will return
 	// 9223372036854775807, the maximum possible share value.
 	WeightedShare int64 `json:"weightedShare"`
+
+	// admissionFairSharingStatus represents information relevant to the Admission Fair Sharing
+	AdmissionFairSharingStatus *AdmissionFairSharingStatus `json:"admissionFairSharingStatus,omitempty"`
 }
+
+type AdmissionFairSharingStatus struct {
+	// ConsumedResources represents the aggregated usage of resources over time,
+	// with decaying function applied.
+	// The value is populated if usage consumption functionality is enabled in Kueue config.
+	ConsumedResources corev1.ResourceList `json:"consumedResources,omitempty"`
+
+	// LastUpdate is the time when share and consumed resources were updated.
+	LastUpdate metav1.Time `json:"lastUpdate,omitempty"`
+}
+
+type AdmissionScope struct {
+	AdmissionMode AdmissionMode `json:"admissionMode,omitempty"`
```

Comment:
```suggestion
	AdmissionMode AdmissionMode `json:"admissionMode"`
```

Could you add API documentation here?

Reply:
And we should probably add the +required tag, missed that.

Reply:
Yes, that makes sense.

Reply:
Added